src/main.cpp

```diff
@@ -18,6 +18,10 @@
 #include "svea_teensy.h"
 
 #include "utility.h"
+#include <stdio.h>
+#include<unistd.h>
+
+SVEA::IMU imu_sensor(nh);
 
 //! Setup ROS
 void rosSetup() {
```

Comment:
Having `negotiateTopics` is strange. Furthermore, having multiple `negotiateTopics` is even stranger. This should be called in `spinOnce`.

Reply:
We removed all the redundant negotiateTopics and now only have one. Common practice is to have one call to negotiateTopics before initializing subscribers/advertisers. I think we added many because of some other issue that has since then been resolved.